eisvogel.tex

```diff
@@ -854,15 +855,18 @@
   $endif$
   \vskip 2em
   \noindent {\Large \textsf{$for(author)$$author$$sep$, $endfor$}}
+  $if(tutor)$
+  \vskip 1em
+  \noindent {\Large \textsf{$for(tutor)$$tutor$$sep$, $endfor$}}
+  $endif$
   \vfill
 }
 $endif$
```

Comment:
Could you revert everything after this line? You accidentally added `$endif$<Paste>`.

Reply:
Sure, I got rid of the accidental `<Paste>`